### PR TITLE
Merge `8.1.x` into  `develop`

### PR DESCRIPTION
**Pull request by tleon**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Merge `8.1.x` into `develop`
| Type?             | improvement
| Category?         | ME
| BC breaks?        |  no
| Deprecations?     | no
| How to test?      | CI & tests 🟢
| UI Tests          | https://github.com/tleon/ga.tests.ui.pr/actions/runs/8643950316
| Sponsor company   | PrestaShop SA

### CODE REVIEW

File: src/PrestaShopBundle/Form/Admin/Product/ProductQuantity.php
Comment (jolelievre) [inline src/PrestaShopBundle/Form/Admin/Product/ProductQuantity.php:1]: I think this file has been removed along with the old product page, it should be cleaned and not be reintroduced in the develop branch

**Review by jolelievre — CHANGES_REQUESTED** [review]

**Review by Progi1984 — DISMISSED** [review]
Ok for UI Tests

**Review by nicosomb — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]

**Review by boherm — DISMISSED** [review]

**Review by M0rgan01 — DISMISSED** [review]

**Comment by nicosomb** [comment]
Don't merge, UI tests are red.

**Comment by nicosomb** [comment]
UI tests: https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/8702817085

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Pull request merged**